substrate/cli/src/informant.rs

```diff
@@ -65,17 +69,27 @@ pub fn start<C>(service: &Service<C>, exit: ::exit_future::Exit, handle: TaskExe
 				Colour::White.paint(format!("{}", best_number)),
 				hash
 			);
+
+			// get cpu usage and memory usage of this process
+			let (cpu_usage, memory) = if sys.refresh_process(self_pid) {
+				let proc = sys.get_process(self_pid).expect("Above refresh_process succeeds, this should be Some(), qed");
+				(proc.cpu_usage(), proc.memory())
+			} else { (0.0, 0) };
+
 			telemetry!(
 				"system.interval";
 				"status" => format!("{}{}", status, target),
 				"peers" => num_peers,
 				"height" => best_number,
 				"best" => ?hash,
-				"txcount" => txpool_status.transaction_count
+				"txcount" => txpool_status.transaction_count,
+				"cpu" => format!("{}%", cpu_usage),
```

Comment:
telemetry should be basic datatypes rather than string-formatted (since it's structured data)

Reply:
Sure. We need to know this cpu usage is percentage based and the memory is kB.

Fixed now.